docs/guide/data_types/timeseries.rst

```diff
@@ -1,16 +1,18 @@
-****************
-TimeSeries Guide
-****************
+.. _timeseries_guide:
+
+**********************
+Time series data guide
+**********************
 
 Time series data are a fundamental part of many data analysis projects in heliophysics as well as other areas.
 **sunpy** provides a TimeSeries object to handle this type of data.
-Much like the `~sunpy.map.Map` object, `~sunpy.timeseries` recognizes data from nine sources and provides instrument-specific information when plotting and similar metadata handling.
-For more information about TimeSeries, and what data sources are currently supported, check out :doc:`/code_ref/timeseries`.
+Much like the `~sunpy.map.Map` object, `~sunpy.timeseries.TimeSeries` can load generic filetypes, and recognizes data from nine specific sources to provide instrument specific data loading and plotting capabilities.
+For more information about TimeSeries, and what file types and data sources are supported, check out :doc:`/code_ref/timeseries`.
```

Comment:
```suggestion
For more information about `~sunpy.timeseries.TimeSeries`, and what file types and data sources are supported, check out :doc:`/code_ref/timeseries`.
```

Reply:
Now, that I'm looking down the list, I see that your subsequent mentions of `TimeSeries` are not linked either. If that's intentional, I'm fine with that and feel free to ignore this suggestion. Perhaps it is too distracting to the eye to have every mention of of the object linked.

Reply:
I didn't really think about this too much, and think I might leave it for this PR, but thanks for catching - I might go through and intentionally link/not-link in another PR.